get_data.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+$username = 'homestead';
+$password = 'secret';
+
+$easy_db = \ParagonIE\EasyDB\Factory::create(
+  'pgsql:host=localhost;dbname=homestead',
+  $username,
+  $password
+);
+
+$socrata = new Socrata('data.medicare.gov');
+//$hospitals = $socrata->get('/resource/rbry-mqwu.json?$limit=10000000');
+
+$bad_ones = $socrata->get('/resource/5u3e-gkdf.json?$limit=100000000');
+
+
+/*
+foreach ($bad_ones as $b) {
+  if (isset($b['mrsa_footnote'])) unset($b['mrsa_footnote']);
+  if (isset($b['ssi_score_footnote'])) unset($b['ssi_score_footnote']);
+  if (isset($b['cauti_score_footnote'])) unset($b['cauti_score_footnote']);
+  if (isset($b['clabsi_score_footnote'])) unset($b['clabsi_score_footnote']);
+  if (isset($b['cdi_footnote'])) unset($b['cdi_footnote']);
+  if (isset($b['domain_2_score_footnote'])) unset($b['domain_2_score_footnote']);
+  if (isset($b['ahrq_psi_90_score_footnote'])) unset($b['ahrq_psi_90_score_footnote']);
+  if (isset($b['domain_1_score_footnote'])) unset($b['domain_1_score_footnote']);
+  if (isset($b['footnotes'])) unset($b['footnotes']);
+  if (isset($b['payment_reduction_footnote'])) unset($b['payment_reduction_footnote']);
+  $easy_db->insert('problems', $b);
+}
+
+$count = 0;
+foreach ($hospitals as $h) {
+  if (isset($h['efficient_use_of_medical_imaging_national_comparison_footnote'])) unset($h['efficient_use_of_medical_imaging_national_comparison_footnote']);
+  if (isset($h['safety_of_care_national_comparison_footnote'])) unset($h['safety_of_care_national_comparison_footnote']);
+  if (isset($h['patient_experience_national_comparison_footnote'])) unset($h['patient_experience_national_comparison_footnote']);
+  if (isset($h['hospital_overall_rating_footnote'])) unset($h['hospital_overall_rating_footnote']);
+  if (isset($h['mortality_national_comparison_footnote'])) unset($h['mortality_national_comparison_footnote']);
+  if (isset($h['readmission_national_comparison_footnote'])) unset($h['readmission_national_comparison_footnote']);
+  if (isset($h['effectiveness_of_care_national_comparison_footnote'])) unset($h['effectiveness_of_care_national_comparison_footnote']);
+  if (isset($h['timeliness_of_care_national_comparison_footnote'])) unset($h['timeliness_of_care_national_comparison_footnote']);
+  $easy_db->insert('hospitals', $h);
+  $count++;
+}
```

Comment:
Both of these funcs are commented out so I wouldn't overwrite data. Maybe do a check to make sure that they don't exist? And if they do, skip?